tests/x11/x3270_ssl.pm

```diff
@@ -0,0 +1,85 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Testopia Case#1595207 - FIPS: x3270
+
+# Summary: x3270 for SSL support testing, with openssl s_server running on local system
+# Maintainer: Wnereiz <wnereiz@eienteiland.org>
+
+use base "x11test";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+
+    my $cert_file     = '/tmp/server.cert';
+    my $key_file      = '/tmp/server.key';
+    my $tracelog_file = '/tmp/x3270-trace.log';
+
+
+    select_console 'root-console';
+
+    # Install x3270
+    zypper_call "install x3270";
+
+    script_run "clear";
+
+    #Generate self-signed x509 certificate
+    type_string
+"openssl req -new -x509 -newkey rsa:2048 -keyout $key_file -days 3560 -out $cert_file -nodes -subj \"/C=CN/ST=BJ/L=BJ/O=SUSE/OU=QA/CN=suse/emailAddress=test\@suse.com\" 2>&1 | tee /dev/$serialdev\n";
+
+    wait_serial "writing new private", 120 || die "openssl req output doesn't match";
+
+    #Workaround for avoiding missing charactors in following commands
+    for (1 ... 20) {
+        send_key "ret";
+    }
```

Comment:
It is a workaround to avoid some characters were not typed. The reason is unknown, I'm not sure if it is only happened on my test environment. I have updated qemu to the latest version in devel.